README.md

```diff
@@ -15,8 +15,6 @@ This repository currently contains:
 
 The following known improvements are not yet implemented:
 
-* Providing our rsync implementation not only tools, but also Go packages for
-  others to use ([draft PR](https://github.com/gokrazy/rsync/pull/3)).
```

Comment:
Am unsure if this whole bullet should be deleted, but at least the link was outdated

Reply:
Yeah, let’s delete the whole bullet. We can track individual improvements in their own GitHub issues, no need to keep this around here :)